blueprints/addon/files/npmignore

```diff
@@ -33,4 +33,7 @@
 # ember-try
 /.node_modules.ember-try/
 /bower.json.ember-try
+/npm-shrinkwrap.json.ember-try
 /package.json.ember-try
+/package-lock.json.ember-try
+/yarn.lock.ember-try
```

Comment:
Is ember-try supposed to clean these up when it is done?

Reply:
It seems it actually does. Same goes for the ones that are already being ignored in fact. I'm wondering if something went wrong locally when shutting down `ember-try` and the clean up didn't happen properly.
Is there maybe a specific reason why e.g. `package.json.ember-try` is already in the ignore files but for instance `yarn.lock.ember-try` isn't? (As it seems both of them are cleaned up afterwards)

Reply:
Ya, both should be included

Reply:
Just ran into an error when shutting down `ember-try` manually. I think this is why I made this PR.

```
Error!
TypeError: Cannot read property 'result' of undefined
TypeError: Cannot read property 'result' of undefined
    at /Users/bertdeblock/Code/open-source/forks/ember-json-viewer/node_modules/ember-try/lib/utils/result-summary.js:18:20
    at Array.forEach (<anonymous>)
    at Class.print (/Users/bertdeblock/Code/open-source/forks/ember-json-viewer/node_modules/ember-try/lib/utils/result-summary.js:17:18)
    at Class._printResults (/Users/bertdeblock/Code/open-source/forks/ember-json-viewer/node_modules/ember-try/lib/tasks/try-each.js:131:54)
    at /Users/bertdeblock/Code/open-source/forks/ember-json-viewer/node_modules/ember-try/lib/tasks/try-each.js:38:14
    at tryCatch (/Users/bertdeblock/Code/open-source/forks/ember-json-viewer/node_modules/rsvp/dist/rsvp.js:525:12)
    at invokeCallback (/Users/bertdeblock/Code/open-source/forks/ember-json-viewer/node_modules/rsvp/dist/rsvp.js:538:13)
    at publish (/Users/bertdeblock/Code/open-source/forks/ember-json-viewer/node_modules/rsvp/dist/rsvp.js:508:7)
    at flush (/Users/bertdeblock/Code/open-source/forks/ember-json-viewer/node_modules/rsvp/dist/rsvp.js:2415:5)
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
```